kv/replica_slice.go

```diff
@@ -103,6 +106,8 @@ func (rs replicaSlice) SortByCommonAttributePrefix(attrs []string) int {
 		return 0
 	}
 	topIndex := len(rs) - 1
+	rand.Seed(time.Now().UTC().UnixNano())
```

Comment:
Committed new one without Seed function and passed rand.Rand out value to both DistSender and replicaSliceTest.